src/protocol/notification/handle.rs

```diff
@@ -437,6 +437,17 @@ impl NotificationHandle {
     pub fn notification_sink(&self, peer: PeerId) -> Option<NotificationSink> {
         self.peers.get(&peer).cloned()
     }
+
+    #[cfg(feature = "fuzz")]
+    /// Expose functionality for fuzzing
+    pub async fn fuzz_send_message(&mut self, command: NotificationCommand) -> crate::Result<()> {
```

Comment:
dq: Instead of adding this method for fuzzing, could we match on the `command` and call the appropriate method on the handler?